packages/addons/driver/hdhomerun/source/resources/language/English/strings.po

```diff
@@ -0,0 +1,54 @@
+# Kodi Media Center language file
```

Comment:
It's be nice to also have the language layout fixed (e.g. ../resources/language/resource.language.en_gb/strings.po)

There's an add-on you can use to automate it all: https://gitlab.com/ronie/script.language-convert

Reply:
tx for hint, currently it is still an 3yr old pr that is heavily outdated and needs rework
if we implement it the one or the other way we will follow the `../resources/language/resource.language.en_gb/strings.po`way 👍